modules/codegen/src/main/scala/com/twilio/guardrail/Common.scala

```diff
@@ -122,8 +122,14 @@ object Common {
     val CodegenDefinitions(clients, servers, supportDefinitions, frameworkImplicits)                     = codegen
     val frameworkImplicitName                                                                            = frameworkImplicits.map(_._1)
 
-    val dtoComponents: List[String]                         = definitions ++ dtoPackage
-    val filteredDtoComponents: Option[NonEmptyList[String]] = if (protocolElems.nonEmpty) NonEmptyList.fromList(dtoComponents) else None
+    val dtoComponents: List[String] = definitions ++ dtoPackage
+
+    // Only presume ...definitions._ import is available if we have
+    // protocolElems which are not just all type aliases.
+    val filteredDtoComponents: Option[NonEmptyList[String]] =
+      NonEmptyList
+        .fromList(dtoComponents)
+        .filter(_ => protocolElems.exists({ case _: RandomType[_] => false; case _ => true }))
```

Comment:
Are there any implications for Java here?

Reply:
This was explicitly to handle the java case 😅 `package-info.java` never seemed/seems to work for me, so this correctly does not emit the `definitons.*` import if the only `protocolElems` we have are just aliases

Reply:
Ah, neat.  We can get rid of `package-info.java` entirely then.